extensions/geode-modules-test/build.gradle

```diff
@@ -22,15 +22,15 @@ apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
 
 
 dependencies {
-  compile(platform(project(':boms:geode-all-bom')))
-  compile(project(':extensions:geode-modules')) {
+  api(platform(project(':boms:geode-all-bom')))
```

Comment:
I ran into issues where using the wrong scope here meant that the bom was not applied across the other scopes. It seems like `api` scope should be more correct but did you verify that the bom is locking down the versions correctly in compile and runtimeOnly scopes?

Reply:
`api` does not lock down the `compile` or `runtimeOnly` scopes. The bom declaration would need to be repeated for `runtimeOnly` dependencies.

We aren't supposed to use `compile` anywhere, so that's ok.

Reply:
Ok, good, so it does lock down `implementation`?

Reply:
Ah, ok. According to [Gradle](https://docs.gradle.org/current/userguide/java_library_plugin.html) the `runtimeOnly` configuration does not inherit constraints from `api`, but *both* are inherited by `runtimeClasspath` which is what hits the POM. At that time, the constraint applies 👍